backend/rag.py

```
@@ -0,0 +1,35 @@
+    QDRANT_URL,
+    QDRANT_API_KEY,
+    QDRANT_COLLECTION,
+    TOP_K
+)
+from typing import List
+
+qdrant = QdrantClient(url=QDRANT_URL, api_key=QDRANT_API_KEY)
+
+def get_embedding(text: str) -> List[float]:
+    """Gera embedding via Ollama (nomic-embed-text)"""
+    response = requests.post(
```

Ruff (0.15.5)

[error] Probable use of `requests` call without timeout

(S113)